pkg/services/store/entity/sqlstash/sql_storage_server_test.go

```diff
@@ -25,113 +25,11 @@ func TestIsHealthy(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestCreate(t *testing.T) {
```

Comment:
I will provide full test coverage in followup PRs. This test is broken and flaky, and is tightly coupled with the previous code. It fails because of several reasons:

- It doesn't contain enough information in the request to make a successful call, thus it fails in the new code where we check several conditions.
- It doesn't use a proper `context.Context`, containing the expected user information. We check this in the new code and we use this information to populate the `CreatedBy` field.

Overall, fixing this test is not worth it, the next PRs will contain tests with real payloads and realistic failure conditions.